bio/gatk/printreadsspark/environment.yaml

```
@@ -0,0 +1,8 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - gatk4 ==4.2.0.0
+  - openjdk =8
+  - snakemake-wrapper-utils ==0.2.0
```

Do these need to be pinned down to these versions, or would newer patch versions also be fine to include (to get bug fixes in). I.e. something like:

```suggestion
  - gatk4 =4.2
  - openjdk =8
  - snakemake-wrapper-utils =0.2
```

Compare to the conda version matching specs:
https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#build-version-spec